openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +475,43 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+# Create Request Endpoint {#create_response_uri}
+
+This endpoint is offered by the Verifier. The Wallet sends a request to this endpoint if the Verifier requests so by passing the `create_request_uri` authorization request parameter. In case of success, the response is a request object that the Wallet MUST process in the same way as a request object as defined in [@RFC9101]. 
+
+Create Request requests MUST be HTTPS POST requests with the "application/json" media type.
```

Comment:
is signed, content type shouldn't be `application/jose`?

Reply:
should be `application/oauth-authz-req+jwt`